app/src/test/java/edu/uc/kansakpb/ikeahjalpare3/FurnitureDataUnitTest.kt

```diff
@@ -63,7 +68,7 @@ class FurnitureDataUnitTest {
     }
 
     private fun thenResultContainsDrawerDresser() {
-        var KopangDrawerDresserFound = false;
+        var KopangDrawerDresserFound = false
```

Comment:
Remove redundant semi-colon as it is not necessary in Kotlin as mentioned in Lecture.